README.md

```diff
@@ -46,26 +69,33 @@ nginx supervisor postgresql libpq-dev npm nodejs-legacy libhdf5-dev libnetcdf-de
 2. Install Python dependencies: `pip install -r requirements.txt`
 
 3. Configurate database permissions for postgresql.  In your `pg_hba.conf`
-   (typically located in `/etc/postgresql/9.5/main`):
 
-Underneath the line
+On MacOS pg_hba.conf can typically be found at:
+ `/usr/local/var/postgres`
 
-```
-local   all             postgres                                peer
-```
+On Linux it can be typically be found at:
+ `/etc/postgresql/9.5/main`
 
-insert
+At the end of the pg_hba.conf file, add the following lines:
```

Comment:
These changes weren't necessarily for me on OS X; @alejandrogarciasalas did you have problems before you inserted them? I think they're only needed on Linux.